src/Entity/Participation.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Entity;
+
+use App\Enum\Workflow\Place\Participation as ParticipationPlace;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ParticipationRepository")
+ */
+class Participation
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Conference", inversedBy="participations")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $conference;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\User", inversedBy="participations")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $participant;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $asSpeaker;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $needTransport = true;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $needHotel = true;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $needTicket = true;
+
+    /**
+     * @ORM\Column(type="json_array")
```

Comment:
```suggestion
     * @ORM\Column(type="json")
```
Utiliser `type="json"`, `json_array` est déprécié